content/events/FilBangkok-DePIN-2024.md

```diff
@@ -0,0 +1,12 @@
+---
+eventName: "FIL Bangkok Main Stage: DePIN Meets AI"
+eventDescription: "​Join us to learn more about key network upgrades, new L2 solutions built on Filecoin, and Filecoin’s future trajectory. Seize this opportunity to help shape the future of Filecoin and contribute to the next chapter of a better internet."
+eventType: "Conference"
+location: "Bangkok, Thailand"
+website: "[https://fil.org/events/](https://lu.ma/aqyqwupe)"
```

Comment:
@laeccles Since we will be able to add only a single website, please confirm which website to use.